docs/index.bs

```diff
@@ -1563,8 +1563,10 @@ spec: storage; urlPrefix: https://storage.spec.whatwg.org/
 
       dictionary RouterCondition {
         URLPatternCompatible urlPattern;
+        RunningStatus runningStatus;
```

Comment:
Adding the new entry to the dictionary. I understood this means `RouterCondition` can have multiple conditions like following:
```js
condition: {
  urlPattern: "/test",
  runningStatus: "running"
}
```

And it makes sense that the SW treats the request as a matched request when all of conditions are satisfied.

Reply:
I have added notes to explain that.